src/agents/regulator.jl

```diff
@@ -1088,6 +1090,30 @@ function solve_agent_problem!(
         # customers.DERGen(h, t, :) .= customers.DERGen_my(reg_year_index, h, t)
     end
 
+    # since regulator problem is ahead of DERA probelm, use previous year's aggregation results.
+    if w_iter >= 2
+        reg_year_dera = model_data.year(first(model_data.index_y)) - 1
+    else
+        reg_year_dera = model_data.year(first(model_data.index_y))
+    end    
+    reg_year_index_dera = Symbol(Int(reg_year_dera))
```

Comment:
This code block has been used multiple times. Perhaps it could be calculated in a function in der_aggregator.jl?

Reply:
Also, is this assuming simulation mode? (Would you want this to be different if you were running in equilibrium mode?)

Reply:
Yes, this is for simulation mode, so bulk power system problem and regulator problem are using the DER aggregation results from previous year. If we're running equilibrium problem this needs to be current year. This needs to be generalized.

Reply:
Elaine to look for a home for this one

Reply:
#9